### PR TITLE
Re-order helm values so the override is last (highest)

### PR DESCRIPTION
**Pull request by chrisghill**

### CODE REVIEW

File: massdriver-application-helm/helm_values.tf [hunk 1]
```diff
@@ -6,7 +6,7 @@ locals {
         "md-deployment-id" = lookup(module.application.params.md_metadata.deployment, "id", "")
       }
     }
-    envs = [for key, val in local.combined_envs : { name = key, value = tostring(val) }]
```
Comment (chrisghill) [inline massdriver-application-helm/helm_values.tf:9]: Not using combined envs anymore (just envs from `massdriver-application`)

File: massdriver-application-helm/main.tf [hunk 2]
```diff
@@ -1,13 +1,12 @@
 
 module "application" {
-  source  = "github.com/massdriver-cloud/terraform-modules//massdriver-application?ref=fc5f7b1"
```
Comment (chrisghill) [inline massdriver-application-helm/main.tf:3]: This and the change on line 10 are paired and rely on this change from 2 weeks ago: https://github.com/massdriver-cloud/terraform-modules/commit/87cc8c237797bb0249140d2e6ba7cbe04822df56

`oidc_issuer_url` is a duplicate field (the value is in the `cluster_artifact` which is already being passed). Bumping the module ref here to get the changes.

File: massdriver-application-helm/main.tf [hunk 3]
```diff
@@ -27,7 +26,7 @@ resource "helm_release" "application" {
     fileexists("${var.chart}/values.yaml") ? file("${var.chart}/values.yaml") : "",
     yamlencode(module.application.params),
     yamlencode(module.application.connections),
-    yamlencode(var.helm_additional_values),
```
Comment (chrisghill) [inline massdriver-application-helm/main.tf:30]: core change

File: massdriver-application-helm/variables.tf [hunk 4]
```diff
@@ -36,14 +36,3 @@ variable "helm_additional_values" {
   type        = any
   default     = {}
 }
-
-variable "additional_envs" {
```
Comment (chrisghill) [inline massdriver-application-helm/variables.tf:40]: No longer needed.

**Review by xpositivityx — APPROVED** [review]

**Pull request merged**